pae-examples.xhtml

```diff
@@ -137,13 +137,13 @@ version-footer: true
                 <div class="highlight"><pre><code id="code_pae4"></code></pre></div>
             </p>
 
-            <h4>Grace notes (accaciature)</h4>
+            <h4>Grace notes (accacciature)</h4>
```

Comment:
Are you sure it is spelled right now?  This website says it is "acciaccatura" (with the "i" in a different place):
https://en.wiktionary.org/wiki/acciaccatura

"acciaccature" is the Italian plural, but in an English context, using "acciaccaturas" would probably be preferrable.